app/assets/javascripts/templates/bulk_buy_modal.html.haml

```diff
@@ -16,10 +16,7 @@
       %button.bulk-buy-add.variant-quantity{type: "button", ng: {click: "add(-1)", disabled: "!canAdd(-1)"}}>
         -# U+FF0D Fullwidth Hyphen-Minus
         －
-      %input.bulk-buy.variant-quantity{
-        type: "number",
-        min: "0",
-        max: "{{ available() }}",
+      %input.bulk-buy.variant-quantity{type: "number", min: "0", max: "{{ available() }}",
```

Comment:
This is just formatting, right?

Reply:
Yep, Rails 5 didn't like it. It threw some fatal errors for "unbalanced brackets" when written like this. :shrug: